neurons/validators/scraper_validator.py

```diff
@@ -465,6 +481,7 @@ def format_response(uid, text):
         try:
             prompt = query["content"]
             tools = query.get("tools", [])
+            tools = ["Google Search", "Youtube Search"]
```

Comment:
is this correct?
recheck this

Reply:
@Chkhikvadze is this correct? Seems odd to overwrite the user supplied tools.